examples_source/1D_simulation(crystalline)/plot_3_satellite_transition_sim.py

```diff
@@ -116,19 +118,19 @@
     magnetic_flux_density=21.14,  # in T
     rotor_frequency=1e9,  # in Hz
     spectral_dimensions=[
-        {
-            "count": 1024,
-            "spectral_width": 1e4,  # in Hz
-            "reference_offset": 1e4,  # in Hz
-            "events": [
-                {
-                    "transition_query": [
+        SpectralDimension(
+            count=1024,
+            spectral_width=1e4,  # in Hz
+            reference_offset=1e4,  # in Hz
+            events=[
+                SpectralEvent(
+                    transition_query=[
```

Comment:
For consistency, add `"ch1"` to the query.

Reply:
added